enterprise/constants.py

```diff
@@ -27,7 +29,9 @@
 )
 
 # Course mode sorting based on slug
-COURSE_MODE_SORT_ORDER = ['verified', 'professional', 'no-id-professional', 'audit', 'honor']
+COURSE_MODE_SORT_ORDER = ['verified', 'professional', 'no-id-professional', 'credit', 'audit', 'honor']
+
+JSON_COURSE_MODE_SORT_ORDER = json.dumps(COURSE_MODE_SORT_ORDER)
```

Comment:
@asadiqbal08 Okay, let's move this to a `json_serialized_course_modes` function in this module.